src/Resources/translations/messages.fr.yml

```diff
@@ -178,6 +178,13 @@ monsieurbiz_advanced_shipping:
             med: 'Points Relais qui proposent la livraison en Point Relais® L'
             24r: 'Point Relais® XL + L + Lockers'
             24l: 'Points Relais qui proposent la livraison en Point Relais® XL '
-            xoh: 'Points Relais compatibles avec la livraison D+1 ' 
+            xoh: 'Points Relais compatibles avec la livraison D+1 '
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏
```suggestion
            xoh: 'Points Relais compatibles avec la livraison D+1'
```